docs/concepts/package-installation-process.md

```diff
@@ -0,0 +1,49 @@
+
+   - If the package is not in the cache, attempt to download it from the sources listed in the configuration. If a package is downloaded, "GET" and "OK" appear in the output.
+
+   - If the package cannot be successfully acquired from any sources, installation fails at this point with an error such as [NU1103](../reference/errors-and-warnings/NU1103.md). Note that errors from `nuget.exe` commands show only the last source checked, but implies that the package wasn't available from any source.
+
+   When acquiring the package, the order of sources in the NuGet configuration may apply:
+
+   - For projects using the PackageReference format, NuGet checks sources local folder and network shares before checking HTTP sources.
+
+   - For projects using the `packages.config` management format, NuGet uses the order of the sources in the configuration. An exception is restore operations, in which case source ordering is ignored and NuGet uses the package from whichever source responds first.
+
+   - In general, the order in which NuGet checks sources isn't particularly meaningful, because any given package with a specific identifier and version number is typically the same on whatever source it's found (this is not guaranteed).
+
+3. (All tools except `nuget.exe`) Save a copy of the package and other information in the *http-cache* folder as described on [Managing the global packages and cache folders](../consume-packages/managing-the-global-packages-and-cache-folders.md).
+
+4. If downloaded, install the package into the per-user *global-packages* folder. NuGet creates a subfolder for each package identifier, then creates subfolders for each installed version of the package.
+
+5. Update other project files and folders:
+
+    - For projects using PackageReference, update the package dependency graph stored in `obj/project.assets.json`. Package contents themselves are not copied into any project folder.
+    - For projects using `packages.config`, copy those parts of the expanded package that match the project's target framework into project's `packages` folder. (When using `nuget install`, the entire expanded package is copied because `nuget.exe` does not examine project files to identify the target framework.)
+    - Update `app.config` and/or `web.config` if the package uses [source and config file transformations](../create-packages/source-and-config-file-transformations.md).
+
+6. Install any down-level dependencies if not already present in the project. This process might update package versions in the process, as described in [Dependency Resolution](../consume-packages/dependency-resolution.md).
```

Comment:
This happens before the assets file generation. I get why we're adding here, but wouldn't it be simpler to say something like NuGet will install the dependencies as required somewhere int he above steps.

Reply:
@nkolev92 Sounds like I should move this (step 6) before step 5?

Reply:
I wouldn't make it a separate step necessarily. 
Something as simple as 'install package and all its dependencies'.